webpack.mix.js

```diff
@@ -11,5 +11,5 @@ const mix = require('laravel-mix');
  |
  */
 
-mix.js('resources/js/app.js', 'public/js')
-    .sass('resources/sass/app.scss', 'public/css');
+mix.copyDirectory('resources/css', 'public/css')
```

Comment:
please use js precessor and sass processor

write js in style of es6 and css in style of scss